docs/diagrams/SortSequenceDiagram.puml

```diff
@@ -0,0 +1,75 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":SortCommandParser" as SortCommandParser LOGIC_COLOR
+participant "s:SortCommand" as SortCommand LOGIC_COLOR
+participant "p:PersonComparator" as PersonComparator LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("sort n/asc")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand("sort n/asc")
+activate AddressBookParser
+
+create SortCommandParser
+AddressBookParser -> SortCommandParser
+activate SortCommandParser
+
+SortCommandParser -> AddressBookParser
+deactivate SortCommandParser
+
+AddressBookParser -> SortCommandParser : parse("n/asc")
+activate SortCommandParser
+
+create SortCommand
+SortCommandParser -> SortCommand
+activate SortCommand
+
+create PersonComparator
+SortCommand -> PersonComparator
+activate PersonComparator
+
+PersonComparator -> SortCommand
+deactivate PersonComparator
+
+SortCommand --> SortCommandParser: s
+deactivate SortCommand
+
+SortCommandParser --> AddressBookParser: s
+deactivate SortCommandParser
+
+AddressBookParser --> LogicManager : s
+deactivate AddressBookParser
+
+LogicManager -> SortCommand : execute()
+activate SortCommand
+
+SortCommand -> Model : sortPerson(p)
+activate Model
+
+Model --> SortCommand
+deactivate Model
+
+create CommandResult
+SortCommand --> CommandResult : result
+activate CommandResult
+
+CommandResult --> SortCommand
+deactivate CommandResult
+
+SortCommand --> LogicManager
+deactivate SortCommand
+destroy SortCommand
```

Comment:
`SortCommand -[hidden]-> LogicManager` to destroy the marker below the end of the activation bar. Looks quite wonky as it is right now

Reply:
Thanks for the solution, I've updated the diagram.